README.md

```diff
@@ -20,36 +20,56 @@
 
 </div>
 
-This is
+## What does `COPIERTemplate` do?
 
-- a [copier](https://copier.readthedocs.io) template/skeleton for Julia packages (see folder [template](template)); and
-- a package that wraps `copier` in Julia using `PythonCall`.
+Creating `Julia` packages involve the creation and edition of many tiny files.
+Wouldn't it be great to automate this?
 
-The template
+This is exactly what `COPIERTemplate` does.
 
-- is opinionated but allows options;
-- can be applied to existing packages (thanks to copier);
-- is automatically reapplied through Pull Requests made by the Copier.yml workflow.
+### FAQ
 
-<!-- agg https://asciinema.org/a/611189 docs/src/assets/demo.gif --speed 2.5 --cols 80 --rows 20 --font-family "JuliaMono" -->
-[![asciicast](docs/src/assets/demo.gif)](https://asciinema.org/a/611189)
+- How is `COPIERTemplate` different from `PkgTemplates`?
+  - it can be applied to existing packages
+  - it invites to follow some (opinionated) best practices
+  - it is automatically reapplied through Pull Requests made by the `Copier.yml` workflow
```

Comment:
```suggestion
  - it can be reapplied to acquire updates made to the template
  - it is automatically reapplied through Pull Requests made by the `Copier.yml` workflow (Work in progress)
```
Updating my list and marking that feature as WIP.